simulator/risc_v/riscv_instr.cpp

```diff
@@ -355,7 +356,10 @@ static const std::vector<RISCVTableEntry<I>> cmd_desc =
     {'B', instr_min,        execute_min<I>,    OUT_ARITHM, ' ', Imm::NO, { Src::RS1, Src::RS2 },  { Dst::RD }, 0, 32 | 64      },
     {'B', instr_minu,       execute_minu<I>,   OUT_ARITHM, ' ', Imm::NO, { Src::RS1, Src::RS2 },  { Dst::RD }, 0, 32 | 64      },
     {'B', instr_gorc,       execute_gorc<I>,   OUT_ARITHM, ' ', Imm::NO, { Src::RS1, Src::RS2 },  { Dst::RD }, 0, 32 | 64      },
+#if 0
     {'B', instr_gorci,      execute_gorci<I>,  OUT_ARITHM, '7', Imm::ARITH, { Src::RS1, Src::ZERO }, { Dst::RD }, 0, 32 | 64   },
+#endif
```

Comment:
I see no point in commenting code out while we can simply remove it. If this instruction is returned, we'll restore it from git.